src/ConsoleApp/DependencyInjection.cs

```diff
@@ -0,0 +1,30 @@
+﻿using BL;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp
+{
+    public class DependencyInjection
+    {
+        public static T ServiceCollectionDI <T>()
```

Comment:
Looks good! Just little renaming needed. Methods are usually named as verbs. Please rename it as `Resolve`.